src/main/kotlin/no/nav/familie/tilbake/api/DokumentController.kt

```diff
@@ -32,15 +33,24 @@ class DokumentController(private val varselbrevService: VarselbrevService,
                          private val henleggelsesbrevService: HenleggelsesbrevService,
                          private val vedtaksbrevService: VedtaksbrevService) {
 
-    @GetMapping("/forhandsvis-manueltVarselbrev/{behandlingId}",
-                produces = [MediaType.APPLICATION_PDF_VALUE])
+    @PostMapping("/forhandsvis/{behandlingId}",
+                 produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
Dette burde være en get. Ingen data som endres av denne operasjonen.

Reply:
Men den er avhengig av ein Dto, og GET skal ikkje motta request-body. Så dermed POST

Reply:
Tror også biblioteket frå felles-frontend, ignorerer body for GET-request

Reply:
Det er fremdeles feil å bruke post. Hvis du fjerner @RequestBody, så behandles feltene i objektet ditt som individuelle quey-parametere.

Reply:
Tror også biblioteket frå felles-frontend, ignorerer body for GET-request.
Og sjekk hentForhåndsvisningVarselbrev(@Valid @requestbody forhåndsvisVarselbrevRequest, den bruker POST når det er ein Dto som sendes frå frontend.

Reply:
Den bruker post for å ikke logge brukers ident. Det er normalt eneste unntaket vi har gjort fra get på det som naturlig er get-requests.